kyverno/manifest.yaml

```diff
@@ -6,3 +6,20 @@ maintainer: "Kyverno Authors"
 description: Kyverno is a policy engine designed for Kubernetes
 url: https://kyverno.io
 category: security
+plans:
+  - label: "Default Policy Security Standard in audit mode"
```

Comment:
should be `Policy Security Standards` (missing `s`) everywhere

Reply:
changed